src/sonarqube-analyzers/sourcemeter-analyzer-base/src/main/java/com/sourcemeter/analyzer/base/core/VersionChecker.java

```diff
@@ -45,7 +45,7 @@ public void onServerStart(Server server) {
         int majorVersion = Integer.parseInt(version[0]);
         int minorVersion = Integer.parseInt(version[1]);
 
-        if (!(majorVersion == 6 && minorVersion >= 3)) {
+        if (!(majorVersion  >= 6 && minorVersion >= 0)) {
```

Comment:
This will accept all of the 6.x versions, but we only supports 6.7+

Reply:
Resolved